compiler/test/compilable/test18238.d

```diff
@@ -0,0 +1,42 @@
+// REQUIRED_ARGS: -preview=bitfields
+// EXTRA_FILES: imports/imp18238.c
+import imports.imp18238;
+
+align(1)        // align means packed
+struct I18238
```

Comment:
Hmm, that's IMO suprising. The *outer* aggregate alignment usually doesn't affect the fields layout at all, just removes any tail padding.

Reply:
Oh well, I guess the remaining 56 bits are treated as tail padding, so probably fine.

Reply:
Maybe, I'll have to have a look again at the state of the `alignment` struct, whether or not `packed` is set true.